osu.Game/Overlays/Comments/DeletedChildrenPlaceholder.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics;
+using osu.Game.Graphics;
+using osu.Framework.Graphics.Sprites;
+using osuTK;
+using osu.Framework.Bindables;
+using Humanizer;
+
+namespace osu.Game.Overlays.Comments
+{
+    public class DeletedChildrenPlaceholder : FillFlowContainer
+    {
+        public readonly BindableBool ShowDeleted = new BindableBool();
+        public readonly BindableInt DeletedCount = new BindableInt();
+
+        private bool canBeShown;
+
+        private readonly SpriteText countText;
+
+        public DeletedChildrenPlaceholder()
+        {
+            AutoSizeAxes = Axes.Both;
+            Direction = FillDirection.Horizontal;
+            Spacing = new Vector2(3, 0);
+            Margin = new MarginPadding { Vertical = 10, Left = 80 };
+            Children = new Drawable[]
+            {
+                new SpriteIcon
+                {
+                    Icon = FontAwesome.Solid.Trash,
+                    Size = new Vector2(14),
+                },
+                countText = new SpriteText
+                {
+                    Font = OsuFont.GetFont(size: 14, weight: FontWeight.Bold, italics: true),
+                }
+            };
+        }
+
+        protected override void LoadComplete()
+        {
+            DeletedCount.BindValueChanged(onCountChanged, true);
+            ShowDeleted.BindValueChanged(onShowDeletedChanged, true);
+            base.LoadComplete();
+        }
+
+        private void onShowDeletedChanged(ValueChangedEvent<bool> showDeleted)
+        {
+            if (canBeShown)
```

Comment:
This is very flimsy. So if showdeleted is set to true it hides it, but if the count is then shown it will go to line 67 which shows it. These cases can all be grouped in an `updateDisplay()` method or similar, and the `canBeShown` bool defined locally.